src/software/ai/navigator/path_planner/enlsvg_path_planner_test.cpp

```diff
@@ -684,6 +684,70 @@ TEST_F(TestEnlsvgPathPlanner, test_going_around_defense_area)
     TestUtil::checkPathDoesNotIntersectObstacle(path.value(), obstacle_polygons);
 }
 
+// TODO (#2913): ENLSVG can not find a path from above defense area to below
+TEST_F(TestEnlsvgPathPlanner, DISABLED_test_going_from_above_enemy_defense_area_to_below)
+{
+    World world = ::TestUtil::createBlankTestingWorld(TbotsProto::FieldType::DIV_B);
+    const Field& field       = world.field();
+    Rectangle navigable_area = field.fieldBoundary();
+
+    // Above defense area
+    Point start{4, 2};
+    // Below defense area
+    Point dest{4, -2};
+
+    std::vector<ObstaclePtr> obstacles =
+        robot_navigation_obstacle_factory.createFromMotionConstraint(
+            TbotsProto::MotionConstraint::INFLATED_ENEMY_DEFENSE_AREA, world);
+
+    EnlsvgPathPlanner planner =
+        EnlsvgPathPlanner(navigable_area, obstacles, ROBOT_MAX_RADIUS_METERS);
+    std::optional<Path> path = planner.findPath(start, dest);
+
+    ASSERT_TRUE(path != std::nullopt);
+
+    // Make sure the start and end points match
+    EXPECT_EQ(start, path.value().front());
+    EXPECT_EQ(dest, path.value().back());
+
+    // Make sure path does not exceed a bounding box
+    Rectangle bounding_box({3, 3}, {4.5, -3});
+    TestUtil::checkPathDoesNotExceedBoundingBox(path.value(), bounding_box);
+    TestUtil::checkPathDoesNotIntersectObstacle(path.value(), obstacles);
+}
+
+// TODO (#2690): Path slightly intersects enemy defense area obstacle
+TEST_F(TestEnlsvgPathPlanner, DISABLED_test_going_from_below_enemy_defense_area_to_above)
```

Comment:
does it graze the inflated enemy defense area or the actual enemy defense area

Reply:
In this test case, the path looks something like this, where it just slightly goes into the obstacle. Not really a huge deal imo, atleast compared to the other disabled test trying to draw the same path in reverse.
![image](https://github.com/UBC-Thunderbots/Software/assets/28585597/094c9cde-150c-4d52-8c98-95a639b9f9ca)

Reply:
Okay in this case, I suggest changing the validation so that it doesn't check if it intersects the obstacle ptr but the actually field enemy defence area itself. The whole point of the inflated defence area is to prevent these resolution discrepancies from the grid-based path plannet

Reply:
Sounds good. I've updated it.